src/coreclr/src/gc/gc.cpp

```diff
@@ -34754,13 +34606,14 @@ void gc_heap::relocate_in_large_objects ()
     }
 }
 
-void gc_heap::mark_through_cards_for_large_objects (card_fn fn,
+void gc_heap::mark_through_cards_for_uoh_objects (card_fn fn,
+                                                    int gen_num,
```

Comment:
nit - spaces need to be adjusted here. the coding convention says that the parameters have the same indentation.